tests/unit/models/reminder-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test } from 'ember-qunit';
```

Comment:
Also a boilerplate test. Either test your model (which you should anyway) or remove this test file.